packages/vm/tests/tester.js

```diff
@@ -182,6 +182,8 @@ function runTests(name, runnerArgs, cb) {
       const runner = require(`./${name}Runner.js`)
       // Tests for HFs before Istanbul have been moved under `LegacyTests/Constantinople`:
       // https://github.com/ethereum/tests/releases/tag/v7.0.0-beta.1
+
+      // TODO: Replace with Common.lteHardfork('Istanbul')
```

Comment:
👍 anything blocking this?

Reply:
no! but I'd rather make in a separate PR, this one is beyond big. It should be handled with care, testing several test scopes.